core/auth/signer.go

```diff
@@ -12,24 +12,26 @@ import (
 	"github.com/ethereum/go-ethereum/crypto"
 )
 
-type signer struct {
+type LocalBlobRequestSigner struct {
 	PrivateKey *ecdsa.PrivateKey
 }
 
-func NewSigner(privateKeyHex string) core.BlobRequestSigner {
+var _ core.BlobRequestSigner = &LocalBlobRequestSigner{}
+
+func NewLocalBlobRequestSigner(privateKeyHex string) *LocalBlobRequestSigner {
```

Comment:
Definitely good to have but would be good to recommend integrators to use something like remote signers while informing them.

Reply:
This would not be a signer with any account balance so probably ok right?

Reply:
Definitely thought of that. But since it could result in DOS or other issues in case it's leaked, better to still suggest a more safer practice.

Reply:
If the key is compromised, anyone can sign a blob request and exhaust the rate limit

Reply:
True, there is always a consequence of a key leaking, but the severity of the consequence can vary. I've been operating under the assumption that we can treat this auth key the way many other web services tend to treat (non-monetary) auth keys, which is as parameters passed in via env vars and CLI params, and managed via K8s and whatever other DevOps infra. 

I think it would delay our goal of having customers use ECDSA handshake auth at launch if we were to ask rollups to set up remote signers infra (or at least configure it with additional new keys), and if we were to build/integrate a remote signer client.

Reply:
Yep, definitely agree. It's not the best to write integrations to many of the cloud options or to pick anyone specific remote signer service. We can make some examples with AWS, GCP and things as fast follow as well.